.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish package to PyPI
+
+on:
+  release:
+    types: [created]
```

Comment:
I would use `published` because of this https://stackoverflow.com/a/61066906/958899